### ISSUE TITLE
TypeError: Cannot assign to read only property 'writeQueueSize' of object '#<TCP>'

### ISSUE THREAD

**Issue by techimist** [devsu/keycloak-nodejs-multirealm#3]
Protecting a url fails with the following error.

```
TypeError: Cannot assign to read only property 'writeQueueSize' of object '#<TCP>'
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:16)
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:18)
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:18)
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:18)
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:18)
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:18)
    at _clone (/Users/techimist/my-project/node_modules/clone/clone.js:162:18)
    at clone (/Users/techimist/my-project/node_modules/clone/clone.js:202:10)
    at NodeCache._wrap (/Users/techimist/my-project/node_modules/node-cache/lib/node_cache.js:674:24)
    at NodeCache.set (/Users/techimist/my-project/node_modules/node-cache/lib/node_cache.js:413:31)
```

Here is a screenshot of call stack which makes more sense:
<img width="300" alt="Screenshot 2020-05-26 at 3 23 09 AM" src="https://user-images.githubusercontent.com/7301265/82845030-73cf4000-9f00-11ea-9f92-140e26a14ea2.png">

`node version: v10.19.0`

**Comment by techimist** [comment]
This [line](https://github.com/devsu/keycloak-nodejs-multirealm/blob/master/src/KeycloakMultiRealm.js#L106) sets the new realm config in `node-cache`.

This works as expected if the session store being used is of type `new session.MemoryStore()` however it fails if I use redis store - `new RedisStore({ client: redisClient });`

The object which needs to be stored in node-cache somewhat looks like this:
```
keykloackObj = {
    authenticated: ƒ authenticated(req)
    config: Config {…}
    deauthenticated: ƒ deauthenticated(req)
    grantManager: GrantManager {…}
    stores: (2) [{…}, SessionStore]
}
```

There are some TCP attributes which are buried very deep in `keykloackObj.stores.SessionStore` which cannot be cloned if the sessionStorage is of type `Redix` (I haven't checked with other storage types.)

**Comment by c3s4r** [comment]
You're right, it looks like an issue when using a different store than memory store.

Right now it would be difficult for me to implement a solution, but if you provide a PR, it would be more than welcome.

**Comment by c3s4r** [comment]
Probably the solution is simple.... just set the `useClones` option to `false` when instantiating `node-cache`.

**Comment by techimist** [comment]
I will check and open a PR.
Thank you 😃

**Comment by c3s4r** [comment]
Fixed in 1.2.0
